src/web/pages/permissions/PermissionDialog.tsx

```diff
@@ -176,13 +210,20 @@ const PermissionDialog = ({
         const resource = isDefined(state.resourceType)
           ? Model.fromElement({name: resourceNameOrId}, resourceTypeOrType)
           : undefined;
+
         let subject;
         if (state.subjectType === 'user') {
-          subject = users.find(user => user.id === state.userId);
+          subject = users.find(
+            user => (user as {id: string}).id === state.userId,
```

Comment:
are the castings really necessary for users, roles and groups find here? afaik all should contain `{id?: string}` already.